merlin/main.py

```diff
@@ -552,6 +562,29 @@ def setup_argparse():
                             Default: ~/.merlin",
     )
 
+    # merlin example
+    example = subparsers.add_parser(
+        "example",
+        help="Generate an example merlin workflow.",
+        formatter_class=RawTextHelpFormatter,
```

Comment:
why does this have this specific formatter class?

Reply:
It was the only way to print with `tabulate` formatting.